test/unit-test/test_memory_functional.c

```diff
@@ -19,6 +19,7 @@
 
 #include <memory/memory.h>
 #include <mock_memory.h>
+#include <random.h>
```

Comment:
I can't see why is this include needed. Could it be unnecessary?

Reply:
random_32_bytes_mcu is used in the same file in _test_attestation_bootloader_hash